app/operators/dcm2nii_operator.py

```diff
@@ -40,11 +39,16 @@ def compute(self, op_input: InputContext, op_output: OutputContext, context: Exe
         for f in input_files:
             shutil.move(os.path.join(input_path, f), dcm_input_path)
 
+        # create output directory for input-ct-dataset.nii.gz
+        dcm_output_path = os.path.join(input_path, 'nii_ct_output')
+        if not os.path.exists(dcm_output_path):
+            os.makedirs(dcm_output_path)
+
         # Run dcm2niix
-        subprocess.run(["dcm2niix", "-z", "y", "-o", input_path, "-f", "input-ct-dataset", dcm_input_path])
+        # subprocess.run(["dcm2niix", "-z", "y", "-o", dcm_output_path, "-f", "input-ct-dataset", dcm_output_path])
 
         # Delete superfluous .json files
-        json_files = glob.glob(input_path + "/*.json")
+        json_files = glob.glob(dcm_output_path + "/*.json")
```

Comment:
I notice you've changed some of the paths, which might explain my other comment saying I couldn't get it working without changing a line 57 in the clinrev_operator.py.

I think we should get your PR working and then we'll do a follow-up to make the pathing more sensible in the MAP. It's a worthwhile improvement in the long run.